### PR TITLE
Fixed `HostPortWaitStrategy` throws `NumberFormatException` when port is exposed but not mapped

### PR DESCRIPTION
**Pull request by barrycommins**
Ports that were exposed but not mapped to a host port were throwing `NumberFormatException` when `getMappedPort()` was called on `ContainerState`.

E.g.
```yaml
redis:
  image: redis
  ports:
    - 6379
```

### CODE REVIEW

File: core/src/test/java/org/testcontainers/junit/DockerComposeExposedPortTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import org.junit.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import java.io.File;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.pass;
+
+public class DockerComposeExposedPortTest {
+
+    @Rule
```
Comment (bsideup) [inline core/src/test/java/org/testcontainers/junit/DockerComposeExposedPortTest.java]: could you please convert it to `try-with-resources`-style and maybe just add to some other Docker Compose test

**Comment by MUHAMMADzmi** [comment]
Balks dad Google

**Comment by bsideup** [comment]
@barrycommins merged, thanks! 👍

**Comment by ilkinulas** [comment]
Are there any plans to release 1.7.1?

**Comment by bsideup** [comment]
@ilkinulas yes, we will do a hotfix release later this week :)

**Pull request merged**